### PR TITLE
[TG-161][Improvement] Server error when save empty device/senario info

### PR DESCRIPTION
**Pull request by Todmy**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/59" target="_blank">ReviewNinja

File: src/app/sensors/details/sensor-detail.html [hunk 1]
```diff
@@ -10,24 +10,49 @@
         <label for="sensor-detail__id">Device name</label>
         <input required [(ngModel)]="sensor.mqttId"
                ngControl="mqttId"
-               type="text" id="sensor-detail__id" placeholder="Enter descriptive name"/>
+               #mqttId="ngForm"
+               type="text" id="sensor-detail__id"
+               placeholder="Enter descriptive name"
+               required/>
```
Comment (kosich) [inline src/app/sensors/details/sensor-detail.html:17]: !fix
required two times

File: src/app/sensors/details/sensor-detail.html [hunk 2]
```diff
@@ -10,24 +10,49 @@
...
+            MQTT ID should be filled
+        </div>
     </div>
     <div class="sensor-detail__type">
         <label for="sensor-detail__id">Type</label>
         <input [(ngModel)]="sensor.type"
                ngControl="type"
-               type="text" id="sensor-detail__type" placeholder="Enter type"/>
+               #type="ngForm"
+               type="text" id="sensor-detail__type"
```
Comment (kosich) [inline src/app/sensors/details/sensor-detail.html]: if you've started splitting each attribute into a new line -- do that for id too

File: src/app/sensors/details/sensor-detail.html [hunk 3]
```diff
@@ -10,24 +10,49 @@
         <label for="sensor-detail__id">Device name</label>
         <input required [(ngModel)]="sensor.mqttId"
                ngControl="mqttId"
-               type="text" id="sensor-detail__id" placeholder="Enter descriptive name"/>
+               #mqttId="ngForm"
```
Comment (kosich) [inline src/app/sensors/details/sensor-detail.html:13]: what is this for?

Comment (Todmy) [inline src/app/sensors/details/sensor-detail.html:13]: discussed personally

File: src/app/sensors/details/sensor-detail.html (hunk 1 above)
Comment (Todmy) [inline src/app/sensors/details/sensor-detail.html:17]: !fixed

File: src/app/sensors/details/sensor-detail.html (hunk 2 above)
Comment (Todmy) [inline src/app/sensors/details/sensor-detail.html]: !fixed

**Comment by kosich** [comment]
⭐

**Pull request merged**